packages/features/portfolio/src/components/portfolio-mobile-nav/portfolio-mobile-nav.tsx

```diff
@@ -41,8 +44,9 @@ const MobileNavContainer = styled(Box)(
   ({ theme }) => `
   display: none;
   ${theme.breakpoints.down('sm')} {
-    background: ${theme.palette.background.accentDefault};
```

Comment:
The divider bar does not match the mock:
![image](https://github.com/user-attachments/assets/4ace4c69-917f-4703-a7ca-202f80def113)
![image](https://github.com/user-attachments/assets/32a724c6-ce0b-485c-adbf-b842194e2c0d)